### DISCUSSION TITLE
Principle: Contributions must be aligned with the purpose of FINOS

### DISCUSSION THREAD

**Discussion by ColinEberhardt** [finos/technical-oversight-committee#15] [General]
Why? The Purpose of FINOS is defined as follows:

> [...] to accelerate collaboration and innovation in financial services through the adoption of open source software, standards and best practices

It is a key statement which captures the scope of the organisation. Alignment with this purpose is the first step towards building a cohesive and relevant ecosystem.

**Comment by jonfreedman** [comment]
I think this basically boils down to the contribution must be something we _think_ will be adopted - do we want to formalise that e.g. a statement of interest from a member organisation that isn't the contributor org?

**Reply by ColinEberhardt** [reply]
> I think this basically boils down to the contribution must be something we think will be adopted

Agreed

> do we want to formalise that e.g. a statement of interest from a member organisation that isn't the contributor org?

The TSC represents a good cross-section of the FINOS membership, I'd hope that this is something we could evaluate ourselves.

**Comment by martin-traverse** [comment]
Hi - addressing this principal and also #16  - I think the wording in the incubation guidelines is quite neat:

_The contributed code or idea has the potential to be useful to end users and organizations in the financial industry_

Perhaps there are some things we can say about that "potential" - does it solve a common problem, will it be easy to adopt, does it interoperate with open and/or de-facto standard technologies/protocols, does it offer good documentation and deployment options, what constraints does it impose.... we can perhaps judge some of these technical criteria more objectively. What actually ends up getting adopted.... will be harder to predict :-)

Of course if a project already has an active community of users or even contributers outside the contributing organisation, then the potential for usefulness answers itself.

On reflection, I do think there is a distinction between usefulness/adoption/interest and alignment more generally with the purpose of FINOS. Real-world usefulness is one element of alignment but I think there are others - community/collaboration being one (which relates to #21) as well as the area of focus - an obvious example dev-ops offerings might not belong in FINOS. Is it fair to say we'd be more interested in solutions that are higher up the stack and tend to be more business-focused in nature?

**Reply by ColinEberhardt** [reply]
Thanks for your feedback, and you're right the quote you took from the [incubation checklist](https://community.finos.org/docs/governance/software-projects/stages/incubating/#incubating-lifecycle-checklist) is an attempt to capture the same intent.

>  I do think there is a distinction between usefulness/adoption/interest and alignment more generally with the purpose of FINOS

That is a very good point. Taking your concrete example:

> an obvious example dev-ops offerings might not belong in FINOS

While this may not be financial services specific, if it is a tool that a number of FINOS members collaborate on, maybe it does achieve the FINOS goal of "collaboration and innovation in financial services"?

Open source collaboration is something that many large organisations struggle with. They may find it easier to collaborate with their peers, in the open, as a first step, before they are happy collaborating more broadly. FINOS potentially provides this 'safe space' for this collaboration to take place.

Whether this is what we want to encourage is up for debate!
